src/main/kotlin/carracing/CarRacingGameManager.kt

```diff
@@ -0,0 +1,29 @@
+package carracing
+
+fun main() {
+    val inputView = InputView()
+    val outputView = OutputView(OutputViewResolver())
+    val numberGenerator = RandomNumberGenerator()
+
+    val carRacingGameManager = CarRacingGameManager(inputView, outputView, numberGenerator)
+    carRacingGameManager.gameStart()
+}
+
+class CarRacingGameManager(
+    private val inputView: InputView,
+    private val outputView: OutputView,
+    private val numberGenerator: NumberGenerator,
```

Comment:
I like that you used an interface.
What are the benefits of using an interface?